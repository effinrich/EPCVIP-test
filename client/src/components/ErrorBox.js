import React from 'react'
import PropTypes from 'prop-types'
import { Box } from 'grommet'

const ErrorBox = ({ children }) => {
  return (
    <Box
      align="center"
      animation="fadeIn"
      background="rgb(202, 25, 0)"
      round="xsmall"
      pad="small"
      margin={{ top: '15px' }}
      pt={3}
    >
      {children}
    </Box>
  )
}

ErrorBox.propTypes = {
  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node])
}

export default ErrorBox
