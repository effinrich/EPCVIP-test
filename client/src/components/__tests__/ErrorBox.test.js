import React from 'react'

import { render } from 'utils/testUtils/helperRtl'

import ErrorBox from '../ErrorBox'

describe('ErrorBox Component', () => {
  it('should render without crashing', () => {
    const { container } = render(<ErrorBox />)
    expect(container).toMatchSnapshot()
  })

  it('should render child element', () => {
    const { container } = render(<ErrorBox />)
    expect(container.firstChild).toBeDefined()
  })
})
