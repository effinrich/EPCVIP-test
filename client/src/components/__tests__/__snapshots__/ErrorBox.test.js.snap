// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBox Component should render without crashing 1`] = `
<div>
  <div
    class="StyledBox-sc-13pk1d4-0 sveFj"
  >
    <span
      class="StyledText-sc-1sadyjn-0 bLeCln"
    />
  </div>
</div>
`;
